python/src/Gibbs_doc.i.in

```diff
@@ -90,6 +97,48 @@ samplers : sequence of :class:`MetropolisHastings`
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::Gibbs::getUpdatingMethod
+"Get how the order of the samplers is determined.
+
+Available values are :
+
+  * 0 (Gibbs.DETERMINISTIC_UPDATING): the samplers are sequentially called in the given order.
+
+  * 1 (Gibbs.RANDOM_UPDATING): one of the samplers is randomly selected to produce the next realization.
+
+Note that under DETERMINISTIC_UPDATING,
+all samplers are called in the specified order every time :meth:`getRealization` is called.
+By contrast, under RANDOM_UPDATING, only one sampler is called
+by :meth:`getRealization`, and this sampler is randomly chosen among all specified samplers.
```

Comment:
maybe just refer to setUpdatingMethod to avoid duplicating the values description,
I think that's what we usually do